proto/ibc/lightclients/solomachine/v2/solomachine.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package ibc.lightclients.solomachine.v2;
 
-option go_package = "github.com/cosmos/ibc-go/v3/modules/light-clients/06-solomachine/types";
+option go_package = "github.com/cosmos/ibc-go/v3/modules/light-clients/06-solomachine;solomachine";
```

Comment:
Similarly to the `07-tendermint` PR. This part is necessary in order to have the go package definition conform to `package solomachine` while existing within the `06-solomachine` directory